docs/dev_guides/core_concepts.md

```diff
@@ -23,16 +23,16 @@ For those who want to integrate custom 3D interactivity into their application,
 They can be activated and deactivated as required.
 
 ### Manipulators
-A **Manipulator** is an object in the scene which allows us to position something. 
+A **Manipulator** is an object in the scene which allows us to position something: a point, a plane, a layer, etc. 
 It is a combination of **translators** and **rotators**. 
 
-Clicking and dragging a **translator** will move the manipulator in the direction of the translator.
-Clicking and dragging a **rotator** will rotate the object.
+Clicking and dragging a **translator** will move the manipulator in the direction of the translator. Translators have a diamond shaped handle.
+Clicking and dragging a **rotator** will rotate the object. Rotators have a ball shaped handle.
 
-<div style="text-align: center;"><img src="https://user-images.githubusercontent.com/7307488/173041374-aec20210-65a7-40a2-bb3d-59f542545b8a.png" alt="A napari-threedee manipulator"  width="30%"></div>
+<div style="text-align: center;"><img src="https://github.com/napari-threedee/napari-threedee/assets/76622105/3edc0809-9ba1-4e23-aae7-5e8817209240" alt="A napari-threedee manipulator"  width="30%"></div>
```

Comment:
```suggestion
<div style="text-align: center;"><img src="https://github.com/napari-threedee/napari-threedee/assets/76622105/3edc0809-9ba1-4e23-aae7-5e8817209240" alt="A napari-threedee manipulator"  width="60%"></div>
```

This will make the image double the size [compared with your other screenshot](https://github.com/napari-threedee/napari-threedee/pull/229#issuecomment-2159687070). If that's not enough, you could bump it to 80% width like Alister suggests.